s5_base/run.sh

```diff
@@ -21,21 +27,19 @@ set -eo pipefail
 samromur_root=/data/asr/samromur/samromur_ldc #samromur
 lm_train=/models/samromur/rmh_2020-11-23_uniq.txt
 prondict_orig=/models/samromur/prondict_rmh_2020-12-02.txt
-prondict=data/prondict_w_samromur.txt
 g2p_model=../preprocessing/g2p/ipd_clean_slt2018.mdl
-stage=0
 
-. utils/parse_options.sh
-
-#TODO: set the samromur data and metadata variables within path.sh
-mfccdir=mfcc
-num_jobs=20
-nj_decode=32
+# Created in this script
+prondict=data/prondict_w_samromur.txt
 localdict=data/local/dict
 
+[ ! -d "$samromur_root" ] && echo "$0: expected $samromur_root to exist" && exit 1;
```

Comment:
I'm against this line since usually I have a sym link to the original data dir so this script would usually fail for me. If david and michal are for it, i'll defer to the general consensus. @Daviderikmollberg